src/components/Subscription.js

```diff
@@ -0,0 +1,46 @@
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+
+const SignupSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+});
+
+const Subscription = () => {
+  return (
+    <div className="mt-[100px] px-3 flex flex-col">
+      <h1 className="text-[#FFFF] mb-2 text-sm">Subscribe to our Newsletter</h1>
+      <Formik
+        className=""
+        initialValues={{
+          email: '',
+        }}
+        validationSchema={SignupSchema}
+        onSubmit={() => {
+          // same shape as initial values
```

Comment:
Do we need this comment here? What you think ? If not, please delete it.

Reply:
done